test/streams_test.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require_relative "helper"
+
+class TestStreams < Minitest::Test
+  include Helper::Client
+
+  def test_read_a_trimmed_entry
```

Comment:
This should go in https://github.com/redis/redis-rb/blob/master/test/lint/streams.rb (that will give you the automatic skip if redis is too old)

Reply:
I renamed the test and moved it with other `xreadgroup` tests ✅